apps/web/pages/styles.css

```diff
@@ -1,11 +1,12 @@
+@import url("https://use.typekit.net/uhm1qvw.css");
```

Comment:
what is this?
please import the font from google fonts
@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');

Reply:
done